package-lock.json

```diff
@@ -2036,7 +2036,7 @@
     },
```

Comment:
I think we are not updating package-lock.json files if we haven’t updated/added npm packages.

Reply:
you are right. This was done by mistake. Will undo it.

Reply:
Done